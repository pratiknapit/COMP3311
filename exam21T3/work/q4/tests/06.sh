psql property -c 'select address(id) from properties where length(address(id)) < 27;'
