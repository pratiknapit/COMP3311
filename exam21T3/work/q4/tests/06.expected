          address           
----------------------------
 56 Cox Street, Coogee 2034
 7 Carr Street, Coogee 2034
 94 Cox Street, Coogee 2034
 31 Bay Street, Coogee 2034
 71 Bay Street, Coogee 2034
 2 Dans Avenue, Coogee 2034
 69 Bay Street, Coogee 2034
(7 rows)
